website/docs/r/aws_auth_backend_config_identity.html.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: "vault"
+page_title: "Vault: vault_aws_auth_backend_config_identity resource"
+sidebar_current: "docs-vault-resource-aws-auth-backend-config-identity"
+description: |-
+  Manages AWS auth backend identity configuration in Vault.
+---
+
+# vault\_aws\_auth\_backend\_config_identity
+
+Manages an AWS auth backend identity configuration in a Vault server. This configuration defines how Vault interacts
+with the identity store. See the [Vault documentation](https://www.vaultproject.io/docs/auth/aws.html) for more
+information.
+
+## Example Usage
+
+```hcl
+resource "vault_auth_backend" "aws" {
+  type = "aws"
+}
+
+resource "vault_aws_auth_backend_config_identity" "example" {
+  backend      = vault_auth_backend.aws.path
+  iam_alias    = "full_arn"
+  iam_metadata = ["canonical_arn", "account_id"]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
```

Comment:
we should also add the `namespace` field here since all resources support setting namespaces. Here's an [example](https://github.com/hashicorp/terraform-provider-vault/blob/main/website/docs/r/kv_secret_v2.html.md?plain=1#L51) of the docs we can add here